CHANGELOG.md

```diff
@@ -95,6 +96,8 @@
 
 ### 💥 Breaking Changes
 
+* `Toolbar` items must be provided as direct children. Wrapping Toolbar items in container
+  components can result in unexpected item overflow.
```

Comment:
Glad we got this in, but note these ended up under the wrong changelog version - fixing now.